src/main/java/uk/gov/hmcts/divorce/solicitor/event/page/SolAboutApplicant1.java

```diff
@@ -34,10 +46,12 @@ public void addTo(final PageBuilder pageBuilder) {
                 .mandatoryWithoutDefaultValue(Applicant::getNameChangedHowOtherDetails,
                 "applicant1NameChangedHow=\"other\"",
                 "If not through marriage or deed poll, please provide details of how they legally changed they name")
-                .optionalWithLabel(Applicant::getGender,
+                .mandatoryWithoutDefaultValue(Applicant::getGender, "divorceOrDissolution=\"dissolution\"",
```

Comment:
divorceOrDissolution field needs to be on the page for the condition  to work

Reply:
you can add readonlynosummary field

Reply:
strange, I ran it through locally and it seemed okay... maybe I missed something. I'll add it